modules/java_api/src/main/java/org/intel/openvino/Any.java

```diff
@@ -6,6 +6,14 @@
 /** This class represents an object to work with different types. */
 public class Any extends Wrapper {
 
+    static {
```

Comment:
Too much code duplication.
As I see it, we can keep this static block only for the Wrapper superclass.

Reply:
Resolved. Thank you.